tasks/oah-install.yml

```diff
@@ -0,0 +1,66 @@
+---
+- name: Ensure old versions of Docker are not installed.
+  package:
+    name:
+      - docker
+      - docker-engine
+    state: absent
+
+- name: Ensure dependencies are installed.
```

Comment:
require sudo privilege

Reply:
Added the required sudo privilege statements